src/commands/general/birthday/birthday.ts

```diff
@@ -1,20 +1,69 @@
+import { dayOptions, monthOptions, userOptions, yearOptions } from '#lib/components/builder';
+import { defaultUserPermissions } from '#lib/types';
 import { Subcommand } from '@kaname-png/plugin-subcommands-advanced';
 import { ApplyOptions } from '@sapphire/decorators';
-import type { ApplicationCommandRegistry } from '@sapphire/framework';
-import { birthdayCommand } from '../../../lib/commands';
+import { type ApplicationCommandRegistry } from '@sapphire/framework';
+import { applyLocalizedBuilder } from '@sapphire/plugin-i18next';
+import { SlashCommandSubcommandBuilder, chatInputApplicationCommandMention } from 'discord.js';
 
 @ApplyOptions<Subcommand.Options>({
 	name: 'birthday',
 })
 export class BirthdayCommand extends Subcommand {
 	public override registerApplicationCommands(registry: ApplicationCommandRegistry) {
 		registry.registerChatInputCommand((builder) => {
-			// It is necessary to call this hook and pass the builder context to register the subcommands stored in the subcommand register in the parent command.
-
-			// Calling both hooks is only necessary if required, it is not mandatory.
-			birthdayCommand(builder);
-
 			this.hooks.subcommands(this, builder);
+
+			return applyLocalizedBuilder(builder, 'commands/birthday:birthday')
+				.setDefaultMemberPermissions(defaultUserPermissions.bitfield)
+				.setDMPermission(false);
 		});
 	}
 }
+
+export const BirthdayApplicationCommandMentions = {
```

Comment:
exporting command mentions, is this the correct place to do it and what is the usecase?